steward/src/commands/orchestrator/start.rs

```diff
@@ -32,7 +32,7 @@ pub struct StartCommand {
 
```

Comment:
Believe we want to update the 'Orchestrator_only' language on line 22 above as well